metrics/node/subscribe_test.go

```diff
@@ -19,7 +19,7 @@ func TestSubscribeServerEvents(t *testing.T) {
 	require.NoError(t, err)
 	err = node.Start()
 	require.NoError(t, err)
-	defer func() { _ = node.Stop() }()
+	defer node.Stop() //nolint: errcheck
```

Comment:
you could do func() {t.NoError}() for every case

Reply:
I'm not sure I understand your comment - you mean something along the lines of:
```
defer func() {
    err := node.Stop()
    require.NoError(err)
}
```